client-firewall-rules.md

```diff
@@ -14,22 +14,69 @@ These endpoints follow the convention of: `[region].data.mcr.microsoft.com`.
 ## Configuring Client Firewall Rules
 
 > ### **DATA ENDPOINT CHANGE**
-> To provide a consistent FQDN between the REST and data endpoints, on **March 3, 2020** the data endpoint will be changing from `*.cdn.mscr.io` to `*.data.mcr.microsoft.com`  
-  Once a change of the data endpoint is confirmed, clients can remove the `*.cdn.mscr.io` FQDN.
+> To provide a consistent FQDN between the REST and data endpoints, beginning **March 3, 2020** the data endpoint will be changing from `*.cdn.mscr.io` to `*.data.mcr.microsoft.com`  
+Depoloyments follow [Azure Safe Deploymnet Pracitces][azure-safe-deployment], which means changes will roll out, per region, over several days.  
+Once a change of the data endpoint is confirmed, clients can remove the `*.cdn.mscr.io` FQDN.
```

Comment:
I think we should not have this line to avoid confusion. Once we go live, we can add this recommendation after confirming full functionality.

Reply:
We want customers to know they can remove these at a future point